### PR TITLE
MP4 Audio & Warnings Fix

### PR DESCRIPTION
**Pull request by gintechsystems**
- MP4 Audio Fixes
- Warnings Fixed (unknown defaults, variable any casting)

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-github "freshOS/Stevia" "4.5.0"
-github "HHK1/PryntTrimmerView" "4.0.0"
```
Comment (NikKovIos) [inline Cartfile:2]: remove this file changes from MR please. Also remove all relying the Carthage.

File: Source/Helpers/YPPermissionCheckable.swift [hunk 2]
```diff
@@ -45,6 +45,8 @@ extension YPPermissionCheckable where Self: UIViewController {
                     block(granted)
                 }
             })
+        @unknown default:
```
Comment (NikKovIos) [inline Source/Helpers/YPPermissionCheckable.swift:48]: Remove `@unknown default` from the MR and merge from last commit from master branch.

File: Source/Pages/Gallery/YPLibraryView.swift [hunk 3]
```diff
@@ -32,7 +32,7 @@ final class YPLibraryView: UIView {
         )
         
         layout(
-            assetViewContainer,
+            assetViewContainer!,
```
Comment (NikKovIos) [inline Source/Pages/Gallery/YPLibraryView.swift:35]: Already fixed in master, remove from MR please

File: YPImagePickerExample/YPImagePickerExample.xcodeproj/xcshareddata/xcschemes/YPImagePickerExample.xcscheme [hunk 4]
```diff
@@ -45,6 +45,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      disableMainThreadChecker = "YES"
```
Comment (NikKovIos) [inline YPImagePickerExample/YPImagePickerExample.xcodeproj/xcshareddata/xcschemes/YPImagePickerExample.xcscheme:48]: Remove from MR

**Review by NikKovIos — CHANGES_REQUESTED** [review]

**Comment by s4cha** [comment]
I took care of the changes while merging, @gintechsystems Thanks a ton for tackling this 👍 😍

**Comment by gintechsystems** [comment]
@s4cha Thank you very much! No problem. This project has helped me a lot with apps have I have building on the side...sorry I couldn't get to fixing these minor edits.

**Pull request merged**